### ISSUE TITLE
Missing plugin image

### ISSUE THREAD

**Issue by benhalverson** [ViennaRSS/viennarss.github.io#27]
Missing plugin image
https://www.vienna-rss.com/development/creating-plugins-for-vienna-2-5/
Are plugins still a thing in 3.5.x

**Comment by josh64x2** [comment]
Yeah they are... that whole page needs going over and converting to markdown etc.

**Comment by matthewruzzi** [comment]
There are more broken links:

> The [AppleScript](http://devworld.apple.com/applescript/) file which, for example, gets information out of Vienna and sends it to an other application.

> For more information on understanding and editing .plists, please go read [this document](http://developer.apple.com/mac/library/documentation/Cocoa/Conceptual/PropertyLists/UnderstandXMLPlist/UnderstandXMLPlist.htm) at the Apple Developer Connection website.


> URL: (mandatory for link-type plugins) this is the URL that is browed when the plugin button is clicked. Vienna substitutes the $...$ placeholders with actual values from the current article being viewed. See [here](http://www.vienna-rss.com/?page_id=65) for a list of placeholders.
